libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -43,10 +43,17 @@ const AP_Param::GroupInfo AP_SerialManager::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("0_BAUD",  0, AP_SerialManager, state[0].baud, AP_SERIALMANAGER_CONSOLE_BAUD/1000),
 
+    // @Param: 0_PROTOCOL
+    // @DisplayName: Console protocol selection
+    // @Description: Control what protocol to use on the console. 
+    // @Values: 1:MAVlink1, 2:MAVLink2
+    // @User: Standard
+    AP_GROUPINFO("0_PROTOCOL",  11, AP_SerialManager, state[0].protocol, SerialProtocol_MAVLink),
+    
```

Comment:
Since this is up here, you might want to follow the others and put it before the baud rate. Or is there any issue with index 0 not being the first?